ui/css/treeherder-info-panel.css

```diff
@@ -3,6 +3,7 @@ div#info-panel {
   font-size: 12px;
   height: 35%;
   max-height: 75%;
+  transform: translateY(0%);
```

Comment:
I actually don't think I need this with the current approach. Will remove it before merge.